packages/publications/src/PublicationFieldTypes.php

```diff
@@ -58,8 +58,8 @@ public static function getRules(self $type): array
             self::Boolean => [new BooleanRule],
             self::Integer => ['integer', 'numeric'],
```

Comment:
I don't think we need numeric here

Reply:
Removing this makes packages/publications/tests/Feature/MakePublicationCommandTest.php:538 fail.

Reply:
Confirmed in a real console that strings cannot be passed as integers:
![image](https://user-images.githubusercontent.com/95144705/212479940-e21cb52f-67b8-482e-b86f-969d5e0bf2cc.png)

False positive?

Reply:
Ah, the message becomes different. e8b22afbb28a03fa403d20f1dae916544e35331f